2200100/문제4.py

```diff
@@ -0,0 +1,22 @@
+T = int(input())
+cnt = 0
+for t in range(T):
+    string = input()
+    string1 = string[::-1]
+    string2 = []
+    string3 = ""
+    for i in string1:
+        if i == 'b':
+            string2.append('d')
+        elif i == 'd':
+            string2.append('b')
+        elif i == 'q':
+            string2.append('p')
+        elif i == 'p':
+            string2.append('q')
+        cnt += 1
+    for k in string2:
+        string3 += k
```

Comment:
좋은 풀이네요! 😄 
```suggestion
    for i in string1:
        if i == 'b':
            string3 += 'd'
        elif i == 'd':
            string3 += 'b'
        elif i == 'q':
            string3 += 'p'
        elif i == 'p':
            string3 += 'q'  
```
`string2` 없이 바로 `string3`을 구하는 방법도 있습니다 👍